src/libmongoc/src/mongoc/mongoc-server-description.c

```diff
@@ -80,14 +80,12 @@ mongoc_server_description_reset (mongoc_server_description_t *sd)
    bson_destroy (&sd->arbiters);
    bson_destroy (&sd->tags);
    bson_destroy (&sd->compressors);
-   bson_destroy (&sd->topology_version);
```

Comment:
Worth mentioning, this was a bug identified by the spec tests. When a server description is marked as Unknown, it should still keep the previous topology version. That way, future state change errors can be ignored. We should only remove the topology version if we see handle another ismaster reply that does not contain it.

Reply:
I don't think so. The addition of topology version is only on master, not released (even in beta). 857d9de942b01ad48b1baa4d5ca2bb0cd05b4c96